iosHackthonApp/.swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+identifier_name:
```

Comment:
I lifted this from the Shell project and then got rid of anything that didn't relate. Let me know if you have any suggestions